Scripts/End/EndPlatformController.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EndPlatformController : MonoBehaviour
+{
+
+    [Space, SerializeField]
+    GameObject endPlatform;
+
+    [Space, SerializeField]
+    ParticleSystem smoke;
+
+    [Space, SerializeField]
+    ParticleSystem fire;
+    Collider player;
```

Comment:
Değişkenin adı, türünün ne olduğunu tam belirtmiyor.

```suggestion
    Collider playerCollider;
```